en/contributing/build.md

```diff
@@ -254,23 +254,32 @@ You can also build the image yourself using the [Dockerfile](https://github.com/
    ```
 
 
-## Install Artifacts
+## Install DroneCore Headers and Static Library
```

Comment:
@julianoes OK, this is updated instructions for installing DroneCore. I've more clearly defined what "install" means, tried to clarify that system install is preferred and the risks, added call for action for Windows, and noted that system wide install requires the `make clean`.